components/brave_extension/extension/brave_extension/webstore.ts

```diff
@@ -5,9 +5,9 @@ const callback = (mutationsList: MutationRecord[], observer: MutationObserver) =
   const buttons: NodeListOf<Element> = document.querySelectorAll('div.webstore-test-button-label')
 
   buttons.forEach((button: Element) => {
-    const text: string = button.innerHTML
+    const text: string = button.textContent || ''
     if (textToMatch.includes(text)) {
-      button.innerHTML = text.replace('Chrome', 'Brave')
+      button.textContent = text.replace('Chrome', 'Brave')
```

Comment:
@keur how do i test that this is working? not sure where the webstore.ts file gets loaded.

Reply:
Open any [chrome extension](https://chrome.google.com/webstore/detail/bookmark-manager/idakfiahffeejfhghndaboolmmhbnepn?hl=en-US) and it should say "Add to Brave" and not "Add to Chrome"

You can also see how these are loaded [here](https://github.com/brave/brave-core/blob/deab19b6b5ec76f20de07ac94cd418df78d22f43/components/brave_extension/extension/brave_extension/manifest.json#L47-L52)

Reply:
Usually there are two buttons, so make sure it also gets the one at the bottom of the page too.